src/octoprint/plugins/virtual_printer/virtual.py

```diff
@@ -14,7 +14,13 @@
 from serial import SerialTimeoutException
 
 from octoprint.plugin import plugin_manager
-from octoprint.util import RepeatedTimer, get_dos_filename, to_bytes, to_unicode
+from octoprint.util import (
+    RepeatedTimer,
+    get_dos_filename,
+    to_bytes,
+    to_unicode,
+    unix_timestamp_to_m20_timestamp,
+)
```

Comment:
`unix_timestamp_to_m20_timestamp` should be imported from `octoprint.util.files`, see other note.